CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+1.4.0 / 2019-08-28
+==================
+
+Bug fixes:
+* Don't use TABs, when triggering `change` JS event upon input value is change (fixes auto-complete control testing in Google Chrome)
+* Fixed inability to manipulate windows when Selenium 3 with Firefox GeckoDriver was used
+* The `clickOnElement` method wasn't working when Selenium 3 with Firefox GeckoDriver was used
+
+Testsuite:
+* Don't test on PHP 5.3 (driver itself would likely continue to work on PHP 5.3 for some time)
+* Adding testing on PHP 7.1, 7.2, 7.3 and 7.4beta
+* PhantomJS was removed from testsuite.
```

Comment:
Please:

*  remove `.` (dot)
* remove `from testsuite`

---

* this was my mistake
* this is list of changes under `Testsuite:` sub-section so they all are about test suite

Reply:
updated